formatTest/unit_tests/expected_output/syntax.rei

```diff
@@ -28,3 +28,9 @@ let myRecordWithFunctions: adders;
  * Public result.
  */
 let result: int;
+
+/* https://github.com/facebook/reason/issues/1614 */
+module Event:
```

Comment:
Ideally I would expect, which might be a bit more work...
```
module Event: (module type of {
  include ReactEventRe;
});
```

Reply:
that'll definitely be a lot more work. 

I can work on that if that's the consensus on how we want this to look like.

Reply:
@IwanKaramazow Is this the same problem as the other function formatting you were working on?

```reason
let a = (
  b,
  c,
) => ...
```

Reply:
It isn't, the function formatting is a separate thing, that needs lots of work.